src/App.js

```diff
@@ -1,24 +1,78 @@
+/* eslint-disable object-curly-newline */
 import React from 'react';
 import './App.scss';
+import classNames from 'classnames';
 
 const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  { id: 1, name: 'Dumplings' },
+  { id: 2, name: 'Carrot' },
+  { id: 3, name: 'Eggs' },
+  { id: 4, name: 'Ice cream' },
+  { id: 5, name: 'Apple' },
+  { id: 6, name: 'Bread' },
+  { id: 7, name: 'Fish' },
+  { id: 8, name: 'Honey' },
+  { id: 9, name: 'Jam' },
+  { id: 10, name: 'Garlic' },
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedId: null,
+  }
+
+  cleaner = () => {
+    this.setState({ selectedId: null });
+  }
+
+  selected = (id) => {
+    this.setState({ selectedId: id });
+  }
```

Comment:
```suggestion
  handleSelect = (id) => {
    this.setState({ selectedId: id });
  }
```
Let's give more meaningful names for such merhods